install/src/deb/control/control

```diff
@@ -0,0 +1,7 @@
+Package: Kylo
+Version: 0.7
```

Comment:
Could this be automatically generated to avoid having to manually change the version number? I know the maven-assembly-plugin has the concept of filters that may be useful here.